app/assets/javascripts/osem-datepickers.js

```diff
@@ -53,31 +49,29 @@ $(function () {
    });
 
    $("#conference-start-datepicker").datetimepicker({
-       pickTime: false,
        useCurrent: false,
        format: "YYYY-MM-DD",
        // conference-start-day >= Current_date
        minDate : today ,
    });
 
    $("#conference-end-datepicker").datetimepicker({
-       pickTime: false,
        useCurrent: false,
        format: "YYYY-MM-DD",
+       minDate: today ,
```

Comment:
Why this?

Reply:
@hennevogel, SInce end date was not getting updated after entering start date so I added it, but then I found that the problem is with js not triggering, then I replaced setMinDate with min date and it started working. ([Reference](https://stackoverflow.com/questions/30536274/setmindate-and-mindate-function-in-bootstrap-3-datetimepicker-not-working))
 
`  $("#conference-start-datepicker").on("dp.change",function (e) {
      $('#conference-end-datepicker').data("DateTimePicker").minDate(e.date);
      if (!$('#conference-end-datepicker').val()) {
         $('#conference-end-datepicker').data("DateTimePicker").setDate(e.date);
      }
  });`

Reply:
should I change it at other places too?

Reply:
@rahul2240 this is no related with fixing the datapickers, right? I think this shouldn't be part of this PR. Let's go step by step.